.circleci/config.yml

```diff
@@ -6,6 +6,43 @@ gpu: &gpu
   environment:
     FPS_THRESHOLD: 900
 commands:
+  build_osx_conda_pkg:
+    description: "Build OSX Conda-Package"
+    parameters:
```

Comment:
It's better to have same parameter signature like [the linux building job](https://github.com/facebookresearch/habitat-sim/blob/f216a1732a2d33747c56df9a4d07485ba70cda3a/.circleci/config.yml#L617-L626):

```
    parameters:
      AIHABITAT_CONDA_CHN:
        type: string
        default: "aihabitat"
      AIHABITAT_CONDA_CHN_PWD_VAR:
        type: string
        default: "AIHABITAT_CONDA_PWD"
      NIGHTLY_FLAG:
        type: string
        default: ""
```
 Maybe, add one more parameter `upload` for cases when no upload required.